presto-native-execution/scripts/release-centos-dockerfile/README.md

```diff
@@ -0,0 +1,387 @@
+# Prestissimo
+
+> 💡 _**PrestoDB** repository: [Presto - https://github.com/prestodb/presto](https://github.com/prestodb/presto)_
+
+> 💡 _**Velox** repository: [Velox - https://github.com/facebookincubator/velox](https://github.com/facebookincubator/velox)_
+
+## Practical Velox implementation using PrestoCpp
+
+> 📝 _**Note:** This readme and the build process was adapted from internal pipeline. You can e-mail the author if you've got questions [milosz.linkiewicz@intel.com](mailto:milosz.linkiewicz@intel.com)_
+
+Prestissimo, marked in PrestoDB GitHub repository as 'presto-native-execution', is effort of making PrestoDB even better using Velox library as a starting point. Both of mentioned - PrestoCpp and Velox - are mainly written using low level `C` and `C++ 17` languages, which makes the build-from-scratch process humongously complicated. To make this process simple, Intel Cloud Native Data Services Team is introducing 3-stage, fully automated Docker build process based on unmodified project GitHub repository.
+
+## Quick Start
+
+### Notice - Pre-required for non-members of Intel CNDS team:
+
+1. Define and export Docker registry, image name and image tag variables
+Notice: Members of Intel CNDS team don't need to define those vars
+
+```bash
+# defaults to 'prestissimo'
+export IMAGE_NAME='PRESTISSIMO_IMAGE_NAME'
+# defaults to 'latest'
+export IMAGE_TAG='PRESTISSIMO_IMAGE_TAG'
+# defaults to ''
+export IMAGE_REGISTRY='MY_DOCKER_REGISTRY'
+# defaults to '0'
+export IMAGE_PUSH='0'
+# defaults to 'avx', more info on Velox GitHub
+export CPU_TARGET="avx"
+```
+2. Clone this repository
+3. Make sure Docker daemon is running in background
+4. Build repo
+
+The process is fully automated and require no interaction for user. The process of building images for the first time can take up to couple of hours (~1-2h using 10 processor cores). `images/prestissimo/build.sh`
```

Comment:
is this folder correct? `imaged/prestissimo/build.sh`? I can't seem to find it on the repo nor the PR.

Reply:
Thanks for your review! Fixed